### PR TITLE
 Fix tests on actions 

### PR DESCRIPTION
**Pull request by chapulina**
Also update the action style.

### CODE REVIEW

**Comment by chapulina** [comment]
+45.62% test coverage at once :smile:

File: .github/ci/after_make.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -l
+
+set -x
+
+# Install (needed for some tests)
+make install
```
Comment (mjcarroll) [inline .github/ci/after_make.sh:6]: Doesn't have to be addressed here, but I think this is resolved in the bazel branch, I'll port it over.

File: src/plugins/CMakeLists.txt [hunk 2]
```diff
@@ -81,10 +81,7 @@ function(ign_gui_add_plugin plugin_name)
     PRIVATE_LINK_LIBS ${ign_gui_add_plugin_PRIVATE_LINK_LIBS} ignition-plugin${IGN_PLUGIN_VER}::register
   )
 
-message(STATUS "ddddddddddddddddddddddddddddddddddddddddd")
```
Comment (mjcarroll) [inline src/plugins/CMakeLists.txt:84]: Beautiful.

File: .github/ci/after_make.sh (hunk 1 above)
Comment (chapulina) [reply .github/ci/after_make.sh:6]: Oh sweet, several libraries need this fix

**Comment by chapulina** [comment]
> were you going to add focal as a platform or is it more work than expected?

We haven't released Blueprint debs for Focal, so we'd need to build dependencies from source. So my plan is to add Focal from Citadel :crossed_fingers:

File: .github/ci/after_make.sh (hunk 1 above)
Comment (mjcarroll) [reply .github/ci/after_make.sh:6]: Hoping to do away with it across the stack :crossed_fingers:

**Pull request merged**